docs/developer-guide/ceps/accepted/cep-002.rst

```diff
@@ -0,0 +1,202 @@
+.. _cep-002:
+
+
+***********************
+CEP 2 - Event Structure
+***********************
+
+* Status: accepted
```

Comment:
Change to 'accepted-in-progress'

Reply:
The "state-machine" in CEP-001 has "accepted / accepted-in-progress" and `"accepted-implemented". 

Since we only have a single PR that will implement it, the main branch is in "accepted" state and once the PR is merged it will be in "implemented" state.

So from that point of view, I think merging this will not mean "in progress", just accepted. Merging a partial implementation would mean in progress and merging a full implementation would mean "implemented".

At least that's my understanding

Reply:
I understand "accepted-in-progress" to mean that an _implementation_ of the suggested enchantment is being worked on, which is the case here.  Do you mean that accepted-in-progress would only happen if you merged the draft PR along with the CEP?

I think it is fine to just include a reference to the draft PR to qualify as "in progress", though we might want to update CEP 1 to spell that out.

Reply:
> Do you mean that accepted-in-progress would only happen if you merged the draft PR along with the CEP?

I would say that "in progress" means that the current main branch has a partial implementation of the CEP.